homeassistant/components/melcloud/config_flow.py

```diff
@@ -0,0 +1,84 @@
+"""Config flow for the MELCloud platform."""
+import asyncio
+import logging
+from typing import Optional
+
+from aiohttp import ClientError, ClientResponseError
+from async_timeout import timeout
+import pymelcloud
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_TOKEN, CONF_USERNAME
+
+from .const import DOMAIN
```

Comment:
This is a known bug in pylint. We'll disable it for now.
```suggestion
from .const import DOMAIN  # pylint: disable=unused-import
```